src/main/java/com/demonwav/mcdev/platform/canary/util/CanaryLegacyColors.java

```diff
@@ -0,0 +1,33 @@
+package com.demonwav.mcdev.platform.canary.util;
+
+import com.demonwav.mcdev.util.CommonColors;
+import org.jetbrains.annotations.NotNull;
+
+import java.awt.Color;
+import java.util.Map;
+
+public final class CanaryLegacyColors {
```

Comment:
Yeah, should just do the apply in the ModuleType. Doesn't need to look exactly like the other ModuleTypes.